R/extract.R

```diff
@@ -65,7 +64,8 @@
 
   # x is executed
   if(is.rasterlyBuild(x)) {
-    .Primitive("[<-")(x, name, value)
+    warning("It is meaningless to replace object from `rasterlyBuild` object", call. = FALSE)
```

Comment:
I'd like to make this warning more explicit -- to be completely honest, I'm not sure what we're reporting here. 

If we are discouraging users from attempting to update `rasterlyBuild` objects in place, I suggest the following change. 

If this is **not** what we're warning about, maybe you can explain briefly in a comment what this is referring to, and we can revise the message further?

```suggestion
    warning("Replacing elements of existing `rasterlyBuild` objects is not supported.", call. = FALSE)
```

Reply:
Thanks!

Reply:
@z267xu Was my understanding correct? If so, feel free to commit the suggestion. If not, we'll want to update the warning message to provide additional context 🙂